app/helpers/concerns/foreman_leapp/hosts_helper_extensions.rb

```diff
@@ -2,13 +2,9 @@
 
 module ForemanLeapp
   module HostsHelperExtensions
-    extend ActiveSupport::Concern
-
-    included do
-      # execute callbacks
+    def multiple_actions
```

Comment:
This would work and we have a way to add new actions to the host menu by registering [helpers](https://github.com/theforeman/foreman/blob/develop/developer_docs/how_to_create_a_plugin.asciidoc#extending-host-ui)

Reply:
Well the reason why I don't use `multiple_actions_provider` is that actions added by this method are expecting to be run in Modal.

But I don't want that, I want to redirect user to Job invocation form after he clicks on link. (same as in `ForemanRemoteExecution::HostsHelperExtensions`)